playbooks/neutron-l3-nonha-to-ha-migration.yml

```diff
@@ -0,0 +1,23 @@
+---
+---
+- hosts: network
+  tasks:
+  - name: get list of all router ids
+    shell: . /root/stackrc; neutron router-list --all-tenants -c id | awk '{print $2}'  | awk 'NF && !/^id$/'
+    register: router_id_list
+    when: neutron.l3ha.enabled|bool
```

Comment:
You've got `network` as the hosts, which means this task would run on BOTH of the network nodes. Either we need to reduce the host set to just one host, or you need to add `run_once: True` to every task so that we only touch each router once.